### DISCUSSION TITLE
QUIC

### DISCUSSION THREAD

**Discussion by Anonym-tsk** [Anonym-tsk/tpws-keenetic#11] [Идеи и предложения]
Пока нет достоверных подтверждений, что перенаправление/блокировка QUIC на что-то влияет.
Поделитесь опытом, есть ли те, у кого без блокировки UDP/443 не работает, а с ней всё хорошо.
Нужно ли добавлять в скрипты правила для QUIC?

**Comment by veagis1** [comment]
Без блокировки quic были зависания периодические приходилось перезагружать приложение или ждать минуты по 2. после запрета udp трафика зависания пропали

**Reply by Anonym-tsk** [reply]
Зависания на Ютубе?

**Reply by veagis1** [reply]
> Зависания на Ютубе?

да, с телевизора и телефона

**Comment by Seamas73** [comment]
Если tpws не работает с udp-пакетами, то, наверное, нужно убрать правило перенаправления udp-пакетов не только для iptables, но и для ip6tables, нет?

**Reply by Anonym-tsk** [reply]
Оно убрано
